src/scripts/deploy_codebuild_source.sh

```diff
@@ -9,4 +9,4 @@ sh $D/../utils/check_mandatory_vars.sh
 # Run this step whenever /source/buildspec.yml or /source/janis-builder.Dockerfile change
 S3_DESTINATION="s3://coa-publisher/janis_builder_factory_source/$DEPLOY_ENV"
 echo "Syncing local source into $S3_DESTINATION"
-aws s3 sync $D/../janis_builder_factory_source $S3_DESTINATION --delete
+pipenv run aws s3 sync $D/../janis_builder_factory_source $S3_DESTINATION --delete
```

Comment:
When our scripts use `pipenv run aws` rather than `aws` then we can be sure that we're using the correct version of awscli that is set by our Pipfile.